gensim/test/test_nmf.py

```diff
@@ -88,7 +88,8 @@ def test_transform(self):
         vec = matutils.sparse2full(transformed, 2)  # convert to dense vector, for easier equality tests
         # The results sometimes differ on Windows, for unknown reasons.
         # See https://github.com/RaRe-Technologies/gensim/pull/2481#issuecomment-549456750
-        expected = [0.03028875, 0.96971124]
+        expected = [0.7723082, 0.22769184]
+        print("vec results", vec)
```

Comment:
We don't want `print` statements in a library. Please remove (here and everywhere).